app/views/recipes/_recipeShow.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="recipe-details">
+
+  <div class="name">
+    <%= recipe.name %>
+    <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+  </div>
+
+  <div class="details">
+    <div class="show-display">
+      <div class = "recipe-time">
+        <p>Preparation time: <%= recipe.preparation_time %> </p>
+        <p>Cooking time: <%= recipe.cooking_time %> </p>
+        <%= render "checkbox", recipe: recipe %>
+      </div>
+    </div>
+    <div class="steps">
+        <%= recipe.description %>
+    </div>
```

Comment:
- [ ] Good job on the beautiful design you have achieved here, however, please follow the given wireframe in terms of the layout of components on the page

Your design
![image](https://github.com/lugard1/recipe_app/assets/81932478/b6bc3451-e6d0-4061-b4a3-ed8b92e85071)

Given design
![image](https://github.com/lugard1/recipe_app/assets/81932478/11f1149b-f10c-45a0-992d-ddaf444f4690)

- Ensure that the `Generate shopping list` and `Add ingredients` buttons are aligned as in the given design.
- Also ensure that instead of the `Destroy this recipe` button you have the button which makes a recipe public when ON and makes it private when OFF as is in the given design.